### PR TITLE
Style post page

### PR DESCRIPTION
**Pull request by paulacarneiro**
- Style post page (post body and post header)
- Create post-info component to be used by post-preview and post-header

### CODE REVIEW

File: components/post-header-info/post-header-info.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import Image from 'next/image';
```
Comment (davelange) [inline components/post-header-info/post-header-info.tsx:1]: Is this component a duplicate of `post-header-data`?

File: components/post-header/post-header.module.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    width: 90%;
+  }
+}
+
+.post-title {
+  width: 86%;
+  margin: 0 auto 1em;
+  font-weight: 300;
+  font-size: 4.375em;
+  text-align: center;
```
Comment (davelange) [inline components/post-header/post-header.module.scss]: From the Figma it looks like the title is aligned left, no center.

**Comment by davelange** [comment]
Screenshot 2022-07-04 at 18 52 24

It looks like some tags aren't being styled (this is a `h1`, from the Productivity post)

**Comment by davelange** [comment]
Screenshot 2022-07-04 at 18 55 05

This header isn't using the right font. Can you check if its font-family is correct?

File: components/post-header-info/post-header-info.tsx (hunk 1 above)
Comment (paulacarneiro) [reply components/post-header-info/post-header-info.tsx:1]: Yes, it wasn't supposed to be there

File: components/post-header/post-header.module.scss (hunk 2 above)
Comment (paulacarneiro) [reply components/post-header/post-header.module.scss]: I forgot to apply the heading styles to `h1`. Regarding the font-family, on Figma the headings inside posts are all 'Studio Feixen Sans', should I change them to use the same font as the paragraphs?

Comment (davelange) [reply components/post-header/post-header.module.scss]: Actually I think Studio Feixen Sans and Studio Feixen are the same font 🤔 so just use the same font-family as the post titles.

**Review by davelange — APPROVED** [review]

**Pull request merged**